master/static/migrations/20210922170815_update-model-registry-get-apis.up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE public.modelsv2 (
+	id integer NOT NULL,
+	name character varying UNIQUE NOT NULL,
+	description character varying,
+	metadata jsonb,
+	creation_time timestamp with time zone NOT NULL,
+	last_updated_time timestamp with time zone,
+    num_versions integer DEFAULT 0 NOT NULL,
+    labels text[],
+    readme text,
+    username text,
```

Comment:
forgot one more thing: can username be a foreign key to the user id in `users` table, similar to how `experiments` table has `owner_id`?